scripts/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+while read -r line; do
+    if [[ $line == *"path"* ]]; then
+        submodule_path=$(echo $line | awk '{print $3}')
+        cd $submodule_path
+		echo "Updating $submodule_path"
+        git pull origin main || exit 1 
+        git checkout $(git tag | tail -n 1)
```

Comment:
@jmoreira-valory 
Please confirm this just in case.
Does this definitely get the latest?Your other one might have been better